ServerCore/Pages/EventSpecific/PH20/ThePlayThatGoesWrong.cshtml

```diff
@@ -854,6 +860,22 @@
                   </div>
                 </td>
               </tr>
+              <tr>
+                <td class="faq-answer-descriptor">
+                  <div class="alert alert-info">
+                    To take it, use the puzzle named Fast Forward
```

Comment:
missing a period perhaps?

Reply:
That's fine; all of the summaries of answers are like this.